### DISCUSSION TITLE
em1 NIC link is down problem

### DISCUSSION THREAD

**Discussion by ldceven** [Security-Onion-Solutions/securityonion#2089] [Unsupported Versions] [answered]
Hello, I encountered a network interface down problem during the installation process. After the final installation is completed, restart the operating system and it will report "NIC link is down"

Server type:
PowerEdge R740xd

Network card information:
Broadcom Adv. Dual 10Gb Ethernet

When troubleshooting, try to plug and unplug the network card, and then the network is restored. Could it be the compatibility of the so system with the network card? thank.

<img width="593" alt="WX20201125-165753@2x" src="https://user-images.githubusercontent.com/67267374/100207901-15345e80-2f43-11eb-825e-aaed4043a061.png">

**Comment by TOoSmOotH** [comment, marked as answer]
I would check your cable. This really has nothing to do with SO as we don't modify the CentOS network stack.
